src/main/java/com/exactpro/th2/inframgr/Config.java

```diff
@@ -32,22 +38,25 @@
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
+import java.nio.file.Path;
 
 public class Config {
```

Comment:
I also believe that we can create this config as a bean

Reply:
Me too, but I wouldn't block this change by this refactoring